libc/src/stdio/CMakeLists.txt

```diff
@@ -163,6 +163,20 @@ add_entrypoint_object(
     libc.src.stdio.printf_core.writer
 )
 
+add_entrypoint_object(
+  asprintf
+  SRCS
+    asprintf.cpp
+  HDRS
+    asprintf.h
+  DEPENDS
+    libc.src.__support.arg_list
+    libc.src.stdio.printf_core.printf_main
+    libc.src.stdio.printf_core.writer
+    libc.src.stdlib.malloc
```

Comment:
this dependency should go on the `vasprintf_internal` target in `printf_core/CMakeLists.txt`. It looks like you need to create that target as well.